acceptance/api_v1_applications_test.go

```diff
@@ -28,22 +28,81 @@ var _ = Describe("Apps API Application Endpoints", func() {
 	})
 
 	Context("Apps", func() {
-		var app1, app2 string
+		Describe("POST /orgs/:org/applications/:app", func() {
+			It("deploys an application with the desired number of instances", func() {
```

Comment:
Maybe these new tests belong to the `apps_test.go` instead?

Reply:
I think this file is meant  to test the API directly, whereas apps_test.go uses the CLI to talk to the API

Reply:
Correct. all the `api_v1_*` files poke directly at the API.

Reply:
So given `makeApp` is using the cli, these tests should be moved. Will do.

Reply:
:bulb: And here I thought from the `It` description that you were directly poking the API.
That should be changed, I believe.
And maybe additional tests which do work directly with the API ?
Around supplication bogus data for instances (negative numbers, non-integer numbers, not-number strings, ...)

Reply:
Done